src/app/shared/object-list/my-dspace-result-list-element/claimed-search-result/claimed-search-result-list-element.component.ts

```diff
@@ -58,4 +60,9 @@ export class ClaimedSearchResultListElementComponent extends SearchResultListEle
     this.workflowitemRD$ = this.dso.workflowitem as Observable<RemoteData<WorkflowItem>>;
   }
 
+  ngOnDestroy() {
+    // This ensures the object is removed from cache, when action is performed on task
+    this.objectCache.remove(this.dso._links.workflowitem.href);
```

Comment:
@ybnd 

unfortunately using the invalidateByHref have a side effect where all the search results are loaded after an action is performed. It's strange because it happens only the first time, I've tried to invastigate without success, so for the moment i'd leave the method as is if you agree. You can see the problem in the video below

https://user-images.githubusercontent.com/2486489/193040254-80c368b4-5fcb-4be8-bb50-10be2f3e96a4.mp4

Reply:
Right, I made a bad call here: it makes sense that this uncaches all entries since invalidate goes after the whole request and not just the object

So it's fine as is, sorry about that.

Not sure why it would be different the second time around though.